core/extrinsics_submission_service/json_transport.hpp

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_EXTRINSICS_SUBMISSION_SERVICE_JSON_TRANSPORT_HPP
+#define KAGOME_CORE_EXTRINSICS_SUBMISSION_SERVICE_JSON_TRANSPORT_HPP
+
+#include <boost/signals2/signal.hpp>
+#include <outcome/outcome.hpp>
+
+namespace kagome::service {
+  class JsonTransport {
```

Comment:
what is the point of this class? I don't get

Reply:
I will use it to test functionality  without real transport.

Reply:
Then please leave a ToDo to implement it later or something, as for now an empty implementation looks very weird

Reply:
Actually you could create mock instead of this class.

Reply:
removed json_transport_impl. added todo, that I'm going to implement transport in next PR